.travis.yml

```diff
@@ -1,15 +1,17 @@
 language: python
-python:
-  - "2.7"
+env:
+  - CONDA="python=2.7"
 before_install:
-  - sudo apt-get update -qq
-  - sudo apt-get install -y python-dev
-  - sudo apt-get install -y libhdf5-serial-dev
-  - sudo apt-get install -y libnetcdf-dev
-  - sudo apt-get install -y python-scipy
-install: 
-  - "pip install numpy"
-  - "pip install netCDF4"
-script: "py.test"
+  - URL=http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh
+  - wget $URL -O miniconda.sh
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - conda update --yes conda
+  - conda config --add channels ioos --force
+  - travis_retry conda create --yes -n test --file requirements.txt $CONDA
```

Comment:
There is a [conda-requirements.txt](https://github.com/pyugrid/pyugrid/blob/master/conda-requirements.txt).  If this is working maybe you could delete the other file in this PR.